velox/common/memory/tests/SharedArbitratorTest.cpp

```diff
@@ -113,7 +113,13 @@ class FakeMemoryOperator : public Operator {
       : Operator(ctx, node->outputType(), id, node->id(), "FakeMemoryNode"),
         canReclaim_(canReclaim),
         allocationCb_(std::move(allocationCb)),
-        reclaimCb_(std::move(reclaimCb)) {}
+        reclaimCb_(std::move(reclaimCb)) {
+    std::pair<MemoryPool*, memory::Allocation*> allocationHook(
+        pool(), &allocation_);
+    TestValue::adjust(
+        "facebook::velox::memory::{}::FakeMemoryOperator::FakeMemoryOperator",
```

Comment:
Why we need testvalue here? If needs, you can pass ctor callback as this is a test operation?